jobs/curator/templates/config/config.yml.erb

```diff
@@ -1,21 +1,7 @@
-<%
-  elasticsearch_hosts = nil
-  elasticsearch_port = nil
-  if_link("elasticsearch") { |elasticsearch_link|
-    elasticsearch_hosts = elasticsearch_link.instances.map { |instance| instance.address }
-    elasticsearch_port = elasticsearch_link.p("elasticsearch.port")
-  }
-  unless elasticsearch_hosts
-    elasticsearch_hosts = p("curator.elasticsearch.hosts")
-  end
-  unless elasticsearch_port
-    elasticsearch_port = p("curator.elasticsearch.port")
-  end
-%>
 ---
 client:
-  hosts: <%= elasticsearch_hosts %>
-  port: <%= elasticsearch_port %>
+  hosts: <%= p("curator.elasticsearch.hosts") %>
```

Comment:
this will require a change in cg-deploy-logsearch

Reply:
Yes, I am going through the config changes now

Reply:
I'm not familiar with this, why would this necessitate a change if the value is being pulled from here?  Or is that not the case?

Reply:
they switched this job config from a "consumes" model to an explicit property: `elasticsearch.hosts`.

Reply:
It looks like it will default to the elasticsearch job running on the same vm. Therefore removing the "consumes" statement should do it.